src/styles/utils/_reset.scss

```diff
@@ -0,0 +1,39 @@
+* {
+  box-sizing: border-box;
+}
+
+html,
+body,
+div,
+span,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+img,
+ol,
+ul,
+li,
+form,
+label,
+main,
+footer,
+header,
+nav,
+section {
```

Comment:
Not all tags from this list are used in the code, so setting styles to them is an unnecessary burden on the code.